src/core/Notification/BaseNotification/BaseNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react';
+import { HtmlDivWithRefProps } from '../../../reset';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  inline: `${baseClassName}--inline`,
+};
+
+export interface BaseNotificationProps extends HtmlDivWithRefProps {
+  /** Style variant. Affects color and icon used.
+   * @default 'neutral'
+   */
+  status?: 'neutral' | 'success' | 'error';
+  /** Set aria-live mode for the alert text content and label.
+   * @default 'assertive'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label for the  alert */
+  labelText?: string;
+  /** Main content of the alert */
```

Comment:
JSDocs seems to mention **Alert** instead of **Notification**.

Reply:
`/** Label for the  alert */`
Two spaces between **the** and **alert**

Reply:
Alert is now fixed, but there is small typo in these: `Notication`, missing the `fi`.

Reply:
Now fixed to be Notification